src/custom/pages/Claim/EligibleBanner.tsx

```diff
@@ -0,0 +1,21 @@
+import { Trans } from '@lingui/macro'
```

Comment:
this is an example (to be kept) to show how it will work. Next PR #2072 goes hard on this